governance/polls/templates/MIP10c14-SP$sp_number: Appoint $appointed_entity as Oracle Light Feed.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Appoint $appointed_entity as Oracle Light Feed (MIP10c14-SP$sp_number) - $date_MONTH_DD,_YYYY
+summary: Signal your support or opposition for appointing $appointed_entity as Oracle Light Feed
+discussion_link: $discussion_link
+vote_type: Plurality Voting
+categories:
+   - Oracles
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Appoint $appointed_entity as Oracle Light Feed (MIP10c14-SP$sp_number) - $date_MONTH_DD,_YYYY
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the $poll_source. This Governance Poll will be active for three days beginning on $date_DAY,_MONTH_DD at 16:00 UTC.
+
+Note: This subproposal has been submitted under the informal weekly cycle rather than the MIP3 Governance Cycle due to inconsistencies between MIP10 and MIP3 and to avoid delaying progress in the oracles domain until MIP10 can be amended.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should abstain.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to appointing $appointed_entity as an Oracle Light Feed.
+
+Please review the application [thread]($discussion_link) to inform your position on this issue before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option, then the following actions will be taken:**
+* This change will be included in a Friday Executive Vote within the next 30 days, absent external factors.
+* If the Executive Vote passes, then this change will become active in the Maker Protocol after the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay) has expired.
+
+**If the votes for the 'No' option equal or exceed the votes for the 'Yes' option, then no further action will be taken at this time.**
+
+---
+
+## Resources
```

Comment:
I think I would just copy over the Resources Section from the generic template

Reply:
Done!